simulation.py

```diff
@@ -143,12 +143,20 @@ def simulate(results_queue, data_all, alpha_settings_path, skip_evaluation):
     pnl_function = TRADING_CONFIGS[settings.target][PNL_FUNCTION](
         OFFICIAL_CURRENCY[region], TRADING_INSTRUMENTS[settings.target])
     results = Simulator(settings, data_required, pnl_function).run()
-    eval_results = (
-        Evaluator().run(TRADING_INSTRUMENTS[settings.target], results)
-        if not skip_evaluation else None)
+    eval_request = {
+        'indicator_names': None,
+        'instruments': TRADING_INSTRUMENTS[settings.target],
+        'data': results}
+    try:
+        eval_results = (
+            requests.post(EVAL_URL, data=pickle.dumps(eval_request)).content
+            if not skip_evaluation else None)
+    except requests.ConnectionError as err:
+        print('Connection falied.', err, sep='\n')
+        eval_results = None
```

Comment:
I think the connection failed may not affect the main process of simulation. Thus, I just notify the user.

Reply:
Can we try 3 times, separated by 1s, before giving up?
Sometimes network is just not stable but the server is actually working.